check_dane

```diff
@@ -81,7 +81,7 @@ def extract_pubkey(cert_binary):
     return pubkey_binary
 
 def validate_dane(cert_binary, tlsa_record):
-    if tlsa_record.usage != 3:
+    if tlsa_record.usage != 3 & tlsa_record.usage != 1:
```

Comment:
Should be "and" instead of the bitwise operator.
Or even simpler:
tlsa_record.usage not in (1, 3)